### PR TITLE
[#RFW-45] Rosetta Android - Jorjin WG7833-B0 Wi-Fi and Bluetooth Integration

### PR DESCRIPTION
**Pull request by vinothraj-aricent**
Cleaned the existing Wi-Fi driver setup, Added the code to execute the hostapd binary for  jorjin driver.

### CODE REVIEW

File: server/SoftapController.cpp [hunk 1]
```diff
@@ -81,24 +77,11 @@ int SoftapController::startSoftap() {
 
     if (!pid) {
         ensure_entropy_file_exists();
-        property_get(DRIVER_VENDOR_NAME, driver_vendor, NULL);
```
Comment (leftbyte) [inline server/SoftapController.cpp]: Is there any reason to remove this detection? It doesn't hurt to keep it in there. Also, this is in the configuration phase so performance isn't really an issue.

File: server/Android.mk [hunk 2]
```diff
@@ -51,6 +51,10 @@ LOCAL_MODULE := netd
 
 LOCAL_INIT_RC := netd.rc
 
+ifdef USES_TI_MAC80211
```
Comment (leftbyte) [inline server/Android.mk]: Are we using this define on certain builds for testing?

File: server/SoftapController.cpp (hunk 1 above)
Comment (Sangilikumar-Aricent) [reply server/SoftapController.cpp]: We are not using driver_vendor_name config field in our board file, so here we are removing the procedure to check the vendor name.

File: server/Android.mk (hunk 2 above)
Comment (Sangilikumar-Aricent) [reply server/Android.mk]: No, Based on this define only, the vendor supported library will be build.

**Pull request merged**